src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/unit_tests/templates/rules/FreemarkerTestRule.kt

```diff
@@ -0,0 +1,107 @@
+package org.vaccineimpact.orderlyweb.tests.unit_tests.templates.rules
+
+import com.gargoylesoftware.htmlunit.html.HtmlPage
+import com.gargoylesoftware.htmlunit.html.HTMLParser
+import freemarker.template.Configuration
+import freemarker.template.Template
+import freemarker.template.TemplateExceptionHandler
+import freemarker.template.Version
+import org.junit.rules.TestRule
+import org.junit.runner.Description
+import org.junit.runners.model.Statement
+import java.io.File
+import java.io.StringWriter
+import java.io.Writer
+import java.lang.IllegalStateException
+import java.net.URL
+import java.util.*
+import javax.xml.transform.Source
+import org.xmlmatchers.transform.XmlConverters.the
+import com.gargoylesoftware.htmlunit.StringWebResponse
+import com.gargoylesoftware.htmlunit.WebClient
+
+
+//This is a simplified kotlinised version of this: https://github.com/Todderz/freemarker-unit-test
+
+class FreemarkerTestRule(val templateName: String, val templatePath: String = "templates",
+                         val includeTemplates: List<String> = listOf()) : TestRule
```

Comment:
I'd say just always include the same templates the app always includes and do away with this var.